Makefile

```diff
@@ -7,9 +7,9 @@ PYTHON_COMMAND := python3
 MODEL_REPO := argmaxinc/whisperkit-coreml
 MLX_MODEL_REPO := argmaxinc/whisperkit-mlx
 
-MODEL_REPO_DIR := ./Models/whisperkit-coreml
-MLX_MODEL_REPO_DIR := ./Models/whisperkit-mlx
-BASE_MODEL_DIR := ./Models
+MODEL_REPO_DIR := ./Sources/WhisperKitTestsUtils/Models/whisperkit-coreml
+MLX_MODEL_REPO_DIR := ./Sources/WhisperKitTestsUtils/Models/whisperkit-mlx
+BASE_MODEL_DIR := ./Sources/WhisperKitTestsUtils/Models
```

Comment:
Would be preferred to keep these models at the top level.

Reply:
the rationale behind it is the this change will allow to simplify the `Package.swift` and the project structure. Models downloaded using `make` are needed basically just for testing, the `WhisperKit's` user will download models to some other location most likely (e.g. the default one provided by `swift-transformers`). Right now we have to provide a custom path and a list of files to exclude in the `Package.swift` just for the sake of having Models directory at the top level.

I'll revert it, this change is not needed for mlx integration